include/common/enum.inc

```diff
@@ -85,605 +85,628 @@ A(Sec_AOT, "aot section")
-O(I64__atomic__rmw8__add_u, 0xFE22, "i64.atomic.rmw8.add_u")
-O(I64__atomic__rmw16__add_u, 0xFE23, "i64.atomic.rmw16.add_u")
-O(I64__atomic__rmw32__add_u, 0xFE24, "i64.atomic.rmw32.add_u")
-O(I32__atomic__rmw__sub, 0xFE25, "i32.atomic.rmw.sub")
-O(I64__atomic__rmw__sub, 0xFE26, "i64.atomic.rmw.sub")
-O(I32__atomic__rmw8__sub_u, 0xFE27, "i32.atomic.rmw8.sub_u")
-O(I32__atomic__rmw16__sub_u, 0xFE28, "i32.atomic.rmw16.sub_u")
-O(I64__atomic__rmw8__sub_u, 0xFE29, "i64.atomic.rmw8.sub_u")
-O(I64__atomic__rmw16__sub_u, 0xFE2A, "i64.atomic.rmw16.sub_u")
-O(I64__atomic__rmw32__sub_u, 0xFE2B, "i64.atomic.rmw32.sub_u")
-O(I32__atomic__rmw__and, 0xFE2C, "i32.atomic.rmw.and")
-O(I64__atomic__rmw__and, 0xFE2D, "i64.atomic.rmw.and")
-O(I32__atomic__rmw8__and_u, 0xFE2E, "i32.atomic.rmw8.and_u")
-O(I32__atomic__rmw16__and_u, 0xFE2F, "i32.atomic.rmw16.and_u")
-O(I64__atomic__rmw8__and_u, 0xFE30, "i64.atomic.rmw8.and_u")
-O(I64__atomic__rmw16__and_u, 0xFE31, "i64.atomic.rmw16.and_u")
-O(I64__atomic__rmw32__and_u, 0xFE32, "i64.atomic.rmw32.and_u")
-O(I32__atomic__rmw__or, 0xFE33, "i32.atomic.rmw.or")
-O(I64__atomic__rmw__or, 0xFE34, "i64.atomic.rmw.or")
-O(I32__atomic__rmw8__or_u, 0xFE35, "i32.atomic.rmw8.or_u")
-O(I32__atomic__rmw16__or_u, 0xFE36, "i32.atomic.rmw16.or_u")
-O(I64__atomic__rmw8__or_u, 0xFE37, "i64.atomic.rmw8.or_u")
-O(I64__atomic__rmw16__or_u, 0xFE38, "i64.atomic.rmw16.or_u")
-O(I64__atomic__rmw32__or_u, 0xFE39, "i64.atomic.rmw32.or_u")
-O(I32__atomic__rmw__xor, 0xFE3A, "i32.atomic.rmw.xor")
-O(I64__atomic__rmw__xor, 0xFE3B, "i64.atomic.rmw.xor")
-O(I32__atomic__rmw8__xor_u, 0xFE3C, "i32.atomic.rmw8.xor_u")
-O(I32__atomic__rmw16__xor_u, 0xFE3D, "i32.atomic.rmw16.xor_u")
-O(I64__atomic__rmw8__xor_u, 0xFE3E, "i64.atomic.rmw8.xor_u")
-O(I64__atomic__rmw16__xor_u, 0xFE3F, "i64.atomic.rmw16.xor_u")
-O(I64__atomic__rmw32__xor_u, 0xFE40, "i64.atomic.rmw32.xor_u")
-O(I32__atomic__rmw__xchg, 0xFE41, "i32.atomic.rmw.xchg")
-O(I64__atomic__rmw__xchg, 0xFE42, "i64.atomic.rmw.xchg")
-O(I32__atomic__rmw8__xchg_u, 0xFE43, "i32.atomic.rmw8.xchg_u")
-O(I32__atomic__rmw16__xchg_u, 0xFE44, "i32.atomic.rmw16.xchg_u")
-O(I64__atomic__rmw8__xchg_u, 0xFE45, "i64.atomic.rmw8.xchg_u")
-O(I64__atomic__rmw16__xchg_u, 0xFE46, "i64.atomic.rmw16.xchg_u")
-O(I64__atomic__rmw32__xchg_u, 0xFE47, "i64.atomic.rmw32.xchg_u")
-O(I32__atomic__rmw__cmpxchg, 0xFE48, "i32.atomic.rmw.cmpxchg")
-O(I64__atomic__rmw__cmpxchg, 0xFE49, "i64.atomic.rmw.cmpxchg")
-O(I32__atomic__rmw8__cmpxchg_u, 0xFE4A, "i32.atomic.rmw8.cmpxchg_u")
-O(I32__atomic__rmw16__cmpxchg_u, 0xFE4B, "i32.atomic.rmw16.cmpxchg_u")
-O(I64__atomic__rmw8__cmpxchg_u, 0xFE4C, "i64.atomic.rmw8.cmpxchg_u")
-O(I64__atomic__rmw16__cmpxchg_u, 0xFE4D, "i64.atomic.rmw16.cmpxchg_u")
-O(I64__atomic__rmw32__cmpxchg_u, 0xFE4E, "i64.atomic.rmw32.cmpxchg_u")
+O(Br_on_null, "br_on_null", 0xD5)
+O(Br_on_non_null, "br_on_non_null", 0xD6)
+// 0xD7 ~ 0xFA: Reserved
+
+// 0xFB prefix - GC Instructions
+OFB(Struct__new, "struct.new", 0xFB, 0)
```

Comment:
Should we also use the hex encoding for the last parameter? I am not sure why the third one uses `hex`, but the last one uses `dec`.

Reply:
Refer to the instruction binary:
https://webassembly.github.io/spec/core/binary/instructions.html#vector-instructions

The second OpCode is a `u32` integer, encoded with LEB-128.
Therefore we use the dec here to present the real decoded `u32` number.

For example, the `f32x4.pmax` instruction, which OpCode is `0xFD 235:u32`, which binary format is `0xFD 0xEB 0x01`.